### ISSUE TITLE
unique constraint failed: folders.pathname

### ISSUE THREAD

**Issue by ollivier262** [sqlitebrowser/sqlitebrowser#2504]
Details for the issue
--------------------

#### What did you see instead?

Useful extra information
-------------------------
> DB4S v3.12.1 [built for x86_64-little_endian-llp64] on Windows 10 (10.0) (winnt/10.0.18363) [x86_64]
> using SQLite Version 3.33.0
> and Qt 5.12.8

**Comment by justinclift** [comment]
Hi, is that an error message you're getting somewhere? :smile:

**Comment by ollivier262** [comment]
I was using dblite for sqlbrowser to correct a DB from xnviewmp. Help sent me to your site and I registered.
I was looking from FAQ or known errors, but it seemed to ask me to post my question.
In parallel, I found my problem: in fact I had another record corresponding to the one after my modification.
I corrected all this and now it is OK.
My modification of xnviewmp DB is now OK, very happy to have tried dblite software.
 
 
I have no other question
 
thank you
 
regards
 
Marcel Ollivier
 
 
 
> Message du 06/12/20 21:56> De : "Justin Clift" > A : "sqlitebrowser/sqlitebrowser" > Copie à : "ollivier262" , "Author" > Objet : Re: [sqlitebrowser/sqlitebrowser] unique constraint failed: folders.pathname (#2504)> >
>
> Hi, is that an error message you're getting somewhere? 😄
—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/sqlitebrowser/sqlitebrowser/issues/2504#issuecomment-739562507", "url": "https://github.com/sqlitebrowser/sqlitebrowser/issues/2504#issuecomment-739562507", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
